core/src/main/java/io/parsingdata/metal/token/Cho.java

```diff
@@ -27,31 +27,31 @@
 
 public class Cho extends Token {
 
-    private final Token[] _tokens;
+    public final Token[] tokens;
```

Comment:
Wow, great catch! I've just turned it into a `return tokens.clone()`. We have to rethink all those data structures at some point anyway, so I would propose to keep it simple for now.